### PR TITLE
torch backend bugfix and speedup ut

### PR DESCRIPTION
**Pull request by WeiweiZhang1**

### CODE REVIEW

File: auto_round_extension/torch/qlinear_torch.py [hunk 1]
```diff
@@ -127,7 +127,6 @@ def pack_248_bits(self, linear, scales, zeros, g_idx=None):
 
         if isinstance(zeros, torch.Tensor):
             zeros = zeros.t().contiguous()
-            zeros -= 1
```
Comment (wenhuach21) [inline auto_round_extension/torch/qlinear_torch.py:130]: help add an accuracy ut for  torch backend  in cpu ut , heng has supported evaluation with limited

Comment (WeiweiZhang1) [reply auto_round_extension/torch/qlinear_torch.py:130]: added in this PR，will update limit args.

File: test/test_cpu/test_torch_backend.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    def __iter__(self):
+        for i in range(2):
+            yield torch.ones([1, 10], dtype=torch.long)
+
+
+class TestAutoRoundTorchBackend(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+        self.model_name = "/models/opt-125m"
```
Comment (wenhuach21) [inline test/test_cpu/test_torch_backend.py]: facebook

File: test/test_cpu/test_torch_backend.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        model = AutoModelForCausalLM.from_pretrained(self.model_name, torch_dtype="auto", trust_remote_code=True)
+        tokenizer = AutoTokenizer.from_pretrained(self.model_name, trust_remote_code=True)
+        bits, group_size, sym = 4, 128, False
+        autoround = AutoRound(
+            model,
+            tokenizer,
+            bits=bits,
+            group_size=group_size,
+            sym=sym,
+            iters=1,
```
Comment (wenhuach21) [inline test/test_cpu/test_torch_backend.py]: using rtn is enough

**Review by wenhuach21 — APPROVED** [review]
